docker/docker-compose-localnet.yaml

```diff
@@ -8,14 +8,15 @@ services:
       - 6060:6060
       - 5002:5002
       - 6002:6002
-      - 7000:7000
+      - 7001:7001
     depends_on:
       - ipfs
       - lotus
     environment:
       - POWD_DEVNET=true
       - POWD_LOTUSHOST=/dns4/lotus/tcp/7777
       - POWD_IPFSAPIADDR=/dns4/ipfs/tcp/5001
+      - POW_GATEWAYHOSTADDR=0.0.0.0:7001
```

Comment:
Just curious why these changes popped up (and in `docker-compose.yaml` as well). Maybe something you needed for local testing? If it's generally useful, feel free to keep the changes.

Reply:
yes, it's just for local testing, would need to revert it back
when I tested locally found that on my laptop some battery management uses this port :)